it/src/main/java/com/google/cloud/teleport/it/datastream/MySQLSource.java

```diff
@@ -0,0 +1,20 @@
+package com.google.cloud.teleport.it.datastream;
+
+public class MySQLSource extends JDBCSource {
+
+    MySQLSource(MySQLSourceBuilder builder) {
+        super(builder);
+    }
```

Comment:
For all these concrete JDBCSource classes (mySQL, postgres, Oracle), I would add some input validation to constructor as I know some of these require certain variables that other don't and they all share the same builder. Checking that all required variable are set will alleviate some pain points with using this RM.